Rasa_Bot/domain.yml

```diff
@@ -19,16 +19,32 @@ intents:
 
 slots:
   ### General
+  # The name of the agent
+  agent_name:
+    type: text
+    initial_value: 'Kris'
+    influence_conversation: false
+  
+  
+  ### User data
+  # The user's age
+  age:
+    type: float
+    min_value: 0.0
+    max_value: 123.0
+    initial_value: 30
+    influence_conversation: false
   # The user's name
   name:
     type: text
     initial_value: ''
```

Comment:
Do you think it would make sense for the `initial_value` to be something like 'UNSETNAME'? I thought that might make it easier (or more obvious) to debug later if the name is not correctly set, than if it's blank.